modules/aws-installing-an-aws-load-balancer-operator.adoc

```diff
@@ -28,67 +28,86 @@ endif::openshift-rosa[]
 cluster.
 * You have installed the
 ifndef::openshift-rosa[]
-{product-title}
+OpenShift CLI (`oc`).
 endif::openshift-rosa[]
 ifdef::openshift-rosa[]
-ROSA
+ROSA CLI (`rosa`).
 endif::openshift-rosa[]
-CLI (`rosa`).
 * You have installed the Amazon Web Services (AWS) CLI.
-* You have installed the OpenShift CLI (oc).
-* You are using OpenShift Container Platform (OCP) 4.13 or later.
+* You are using OpenShift Container Platform 4.13 or later.
 
 [IMPORTANT]
 ====
-When installing an ALBO for use with
+When installing an AWS Load Balancer Operator for use with
 ifndef::openshift-rosa[]
 a {product-title}
 endif::openshift-rosa[]
 ifdef::openshift-rosa[]
 a ROSA
 endif::openshift-rosa[]
-cluster in an AWS Local Zone (LZ), you must enable the AWS LZ for the account, and AWS Elastic Load Balancing v2 (ELBv2) services must be available in the AWS LZ.
+cluster in an AWS Local Zone (LZ), you must enable the AWS Local Zone for the account. Additionally, you must ensure that AWS Elastic Load Balancing v2 (ELBv2) services exist in the AWS Local Zone.
 ====
+
 .Procedure
 
 . Identify the cluster infrastructure ID and the cluster OpenID Connect (OIDC) DNS by running the following commands:
++
 .. Identify the
 ifndef::openshift-rosa[]
-{product-title}
+{product-title} cluster ID:
++
+[source,terminal,subs="quotes,verbatim"]
+----
+$ rosa describe cluster --cluster=<cluster_name> | grep -i '^ID'
```

Comment:
Watch out here - a non-ROSA cluster won't use the "rosa describe" command, probably the "oc describe" command instead?

Reply:
Good catch. Updated!